lib/tasks/migrate.js

```diff
@@ -56,6 +56,12 @@ module.exports = function runMigrations(context) {
                 message: 'Invalid database username or password',
                 environment: context.instance.system.environment
             });
+        } else if (error.stdout && error.stdout.match(/npm install sqlite3 \-\-save/)) {
+            // We check stdout because knex outputs to stdout on this particular error
+            error = new errors.SystemError(
```

Comment:
How come SystemError doesn't take an Object, but ConfigError does?

BTW: I'm working on a docs update to link to here 2 mins